### PR TITLE
Fix minute calculation for a running timer

### PR DESCRIPTION
**Pull request by kaylakremer**
This is in response to an [issue](https://github.com/harvesthq/harvest-react-native/issues/1611) brought up on our mobile repo.

With a running timer, when the time changes from `1:59` to `2:00` for example, the time briefly displays at `1:00` instead of as `2:00` until another minute goes by, reverting back to the correct display time of `2:01`.

This is due to the use of `Math.round` which causes the minutes to display as `:00` a bit too soon before the hour calculation increases by 1 since it will round up when the remainder value is 59.5 or greater.

I added an `if` conditional to instead use `Math.floor` when the remainder value is between 59.5 and 60 to keep the value displaying as `:59` until the hour value increases by 1.

### CODE REVIEW

**Comment by pusewicz** [comment]
Could we please add a test to cover this scenario? Thanks!

**Comment by KaylaKremer** [comment]
@pusewicz Sure, I can add it to our mobile codebase once this is reviewed and approved as being the appropriate fix for this edge case.

**Review by braddunbar — APPROVED** [review]
Works for me! We should probably port this to clockwork and Harvestapp.

**Review by pusewicz — APPROVED** [review]

**Comment by pusewicz** [comment]
@KaylaKremer Thank you!

File: src/index.ts [hunk 1]
```diff
@@ -33,7 +33,12 @@ export const toHHMM = (input?: number | string): string => {
   const sign = total < 0 ? '-' : ''
   total = Math.abs(total)
   const hours = Math.floor(total / 60)
-  const minutes = Math.round(total) % 60
+  let minutes = total % 60
+  if (minutes >= 59.5 && minutes < 60) {
+    minutes = Math.floor(minutes)
```
Comment (evanwalsh) [inline src/index.ts:38]: FWIW, we _could_ just hardcode `59` here since it's a known value.

**Review by evanwalsh — APPROVED** [review]
